types/marked-terminal/package.json

```diff
@@ -5,6 +5,7 @@
     "projects": [
         "https://github.com/mikaelbr/marked-terminal"
     ],
+    "minimumTypeScriptVersion": "4.7",
```

Comment:
Ouch, this is a bummer for us wanting to node16 the whole repo

Reply:
Not everything will require this. Honestly I didn’t look super closely at why this was needed here, but I figured it wasn’t much of a loss since this package uses `exports` anyway. But now I’m hitting a minimum of 5.0 needed for something in `marked` typings, so I might take a closer look and ask for an older version of `marked` for testing if I can, since the range is quite large.

Reply:
The CI failure was happening because without understanding `module: node16`, the TS program was falling back to `moduleResolution: classic`. But specifying `moduleResolution` is not currently allowed. Losing my mind with this PR.